mypy/test/data.py

```diff
@@ -17,6 +17,11 @@
 
 root_dir = os.path.normpath(PREFIX)
 
+# Files to run with new semantic analyzer.
+new_semanal_files = [
```

Comment:
It would be cleaner to move this definition into a separate module. Also add comment mentioning that this is a temporary hack and will be removed after we've got all test cases working with the new semantic analyzer.